buildenv/jenkins/tapVerification/tapVerification.groovy

```diff
@@ -0,0 +1,75 @@
+#!groovy
+
+def AQA_REPO = params.AQA_REPO ?: "adoptium"
+def AQA_BRANCH = params.AQA_BRANCH ?: "master"
+def JDK_VERSIONS = params.JDK_VERSIONS.trim().split("\\s*,\\s*");
+def PLATFORMS = params.PLATFORMS.trim().split("\\s*,\\s*");
+def TARGETS = params.TARGETS.trim().split("\\s*,\\s*");
+def LABEL = (params.LABEL) ?: ""
+def LABEL_ADDITION = (params.LABEL_ADDITION) ?: ""
```

Comment:
- https://ci.adoptopenjdk.net/view/Test_grinder/job/TAP_verification/11/console does not work because this PR has not been merged. `aqaTap.sh` does not exist.
- `TARGETS` is intended for future use. I would like users to provide what TARGET they want to check, not just be limited to the predefined AQA 9 targets.
- yes, `LABEL_ADDITION` is not needed. It should be removed.

Reply:
I have pointed the main config to your branch (where aqaTap.sh exists), but realize I need to run with CUSTOMIZED_SDK_URL_CREDENTIAL_ID.